samples/nrf9160/modem_shell/README.rst

```diff
@@ -16,6 +16,8 @@ The sample supports the following development kits:
 
 .. table-from-sample-yaml::
 
+.. include:: /includes/external_flash_nrf91.txt
```

Comment:
I'll comment here but applies to all Requirements sections.
When looking at the generated document, it's starting to look like there could be subsections or something to distinguish between supported development kits, external flash, and TFM.

Reply:
I believe paragraph break after each requirements (development kits, external flash, and TFM) and now moving the external flash content after TFM, makes the document more readable.

Reply:
The problem with paragraph breaks is that you'll have similar breaks for the note and numbered list.

Anyway, I'll leave this to the documentation team to decide. Maybe something you can discuss with your colleagues and address with separate PRs if needed.

Reply:
Your plan sounds good.